pkg/rtc/mediatrackreceiver.go

```diff
@@ -74,16 +73,14 @@ func (m mediaTrackReceiverState) String() string {
 
 type simulcastReceiver struct {
 	sfu.TrackReceiver
-	priority   int
-	layerSSRCs [livekit.VideoQuality_HIGH + 1]uint32
```

Comment:
Applying these in `trackInfo` directly on write. Previously, there were fields like these and they were all patched in on read. Changing it to apply directly on write and propagate.